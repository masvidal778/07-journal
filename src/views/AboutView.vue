<template>
  <div class="about">
    <h1>This is an about page</h1>
  </div>
</template>

//about.spec.js
//describe( Proves a l'AboutView )
//test() ha de renderitzar el component correctament
//ha de fer match a l'snapshot

/src/views/