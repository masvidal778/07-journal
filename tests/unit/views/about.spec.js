import AboutView from "@/views/AboutView.vue";
import { shallowMount } from "@vue/test-utils";

describe("proves a l'About View", () => {

    test("Renderitza correctament el component", () => {
        const wrapper  = shallowMount(AboutView)
        expect( wrapper.html() ).toMatchSnapshot()
    })
})