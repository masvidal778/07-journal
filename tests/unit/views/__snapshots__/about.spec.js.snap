// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`proves a l'About View Renderitza correctament el component 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;
